src/www/services_rfc2136_edit.php

```diff
@@ -58,6 +58,9 @@
     $pconfig['usetcp'] = isset($a_rfc2136[$id]['usetcp']);
     $pconfig['usepublicip'] = isset($a_rfc2136[$id]['usepublicip']);
 
+    $pconfig['recordtype'] = $a_rfc2136[$id]['recordtype'];
+    if (!$pconfig['recordtype']) $pconfig['recordtype'] = "both";
```

Comment:
@fichtner I cannot find this in the tools repository as a documentation, please add this tip for other contributors.